### DISCUSSION TITLE
Disable "checkable" prop but I still want to effect parent's check situation.

### DISCUSSION THREAD

**Discussion by Mert18** [ant-design/ant-design#43791] [General]
### What problem does this feature solve?

When "checkable" prop is true, It does not effect the parent. But in my case, as seen in the img: https://imgur.com/a/nIDKHtk , Parent node has one group node, and others are children. Children are not checkable, thus when I check the group node, the parent node is checked. But I do not want it. I want my parent node to be half-checked in this situation. "checkStrictly" is not the solution because the parent is not even half-selected when I select the group node. How can I find a way to achieve my goal? Thank you in advance.

### What does the proposed API look like?

Is there a way to manually half-check parents without knowing the node names beforehand? Or Is there already a built-in function to achieve my goal?

**Comment by zombieJ** [comment]
antd use `rc-tree` conduction check logic here: https://github.com/react-component/tree/blob/master/src/utils/conductUtil.ts

If you want to diff check logic, you can follow the code and modify. But still need to use `checkStrictly` to handle it self.
